attribute/value.go

```diff
@@ -231,15 +231,18 @@ func (v Value) Emit() string {
 	case BOOL:
 		return strconv.FormatBool(v.AsBool())
 	case INT64SLICE:
-		return fmt.Sprint(v.asInt64Slice())
+		j, _ := json.Marshal(v.asInt64Slice())
```

Comment:
I'm currently ignoring those errors because importing `go.opentelemetry.io/otel` creates an import cycle.

```
package go.opentelemetry.io/otel/attribute
        imports go.opentelemetry.io/otel
        imports go.opentelemetry.io/otel/internal/global
        imports go.opentelemetry.io/otel/attribute: import cycle not allowed
```

It doesn't look like anything else in attributes has to handle errors.
Also, we control the format of those attributes, so I don't see any case where they could generate errors.

Reply:
In other transforms we have defaulted to responding with something like `fmt.Sprintf("invalid: %v", v.asInt64Slice())`. That way the user has some indication the error occurred.